feature-libs/asm/package.json

```diff
@@ -29,6 +29,7 @@
     "@angular/common": "^15.2.4",
     "@angular/core": "^15.2.4",
     "@angular/forms": "^15.2.4",
+    "@angular/router": "^15.2.4",
```

Comment:
If considering adding a new dependency, please discuss the requirements with @Oezak, Levent for approval.

Wiki entry: https://wiki.one.int.sap/wiki/pages/viewpage.action?spaceKey=spar&title=%5BDRAFT%5D+Guideline%3A+Adding+3rd+party+dependencies

Reply:
Fixed with using existing component